plugins/woocommerce/changelog/fix-meta-data-customer-details

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Do not send user meta data back in .
```

Comment:
Possibly:

```suggestion
Do not send user meta data back in `woocommerce_get_customer_details` response.
```

Reply:
Oops, looks like an issue with automatic changelog creation, thanks for noticing 😄